docs/whats_new/v0.6.2.md

```diff
@@ -0,0 +1,40 @@
+# Version 0.6.2
+
+v0.6.2 is a security and bug fix release.
+
+This release is backward compatible to 0.6.0 and 0.6.1. As always, only load previously trained models using the same version of AutoGluon that they were originally trained on.
+Loading models trained in different versions of AutoGluon is not supported.
+
+See the full commit change-log here: https://github.com/autogluon/autogluon/compare/v0.6.1...v0.6.2
+
+This version supports Python versions 3.7 to 3.9. 0.6.x are the last releases that will support Python 3.7.
+
+# Changes
+
+## Documentation improvements
+
+- Ray usage FAQ (#2559) - @yinweisu
+- Fix missing Predictor API doc (#2573) - @gidler
+- 2023 Roadmap Update (#2590) - @lnnixma
+- Image classifiction tutorial update for bytearray (#2598) - @suzhoum
+- Fix broken tutorial index links (#2617) - @shchur
+- Improve timeseries quickstart tutorial (#2653) - @shchur
+
+
+## Bug Fixes / Security
+
+- [multimodal] Refactoring and bug fixes(#2554, #2541, #2477, #2569, #2578, #2613, #2620, #2630, #2633, #2635, #2647, #2645, #2652, #2659) - @zhiqiangdon, @yongxinw, @FANGAreNotGnu, @sxjscience, @lnnixma
```

Comment:
@lnnixma -> @innixma

I not L (I wanted to change the username to all lowercase, but for whatever reason GitHub allows you to change your alias to something else, but not to the same name with different casing...)

Reply:
Didn't notice. Fixed